web-react-ts/src/pages/directory/quick-facts/QuickFactsQueries.ts

```diff
@@ -1,6 +1,6 @@
 import { gql } from '@apollo/client'
 
-export const FELLOWSHIP_AVG_WEEKDAY_ATTENDANCE_THIS_MONTH = gql `
```

Comment:
This change of yours is going to lead to merge conflicts

Reply:
I have no idea what I was doing there. Will check please.